rasa/nlu/extractors/mitie_entity_extractor.py

```diff
@@ -89,16 +89,19 @@ def train(
         if found_one_entity:
             self.ner = trainer.train()
 
-    def _prepare_mitie_sample(self, training_example) -> Any:
+    def _prepare_mitie_sample(self, training_example: Message) -> Any:
         import mitie
 
         text = training_example.text
         tokens = training_example.get(TOKENS_NAMES[TEXT_ATTRIBUTE])
-        sample = mitie.ner_training_instance([t.text for t in tokens])
+        tokens_without_cls = tokens[:-1]
```

Comment:
in crf extractor, we add `[:-1]` when we read tokens, I guess it makes sense to do the same for consistency, with a comment